README.md

```diff
@@ -9,6 +9,11 @@ If you have questions, visit us on IRC in [#opensuse-factory](irc://chat.freenod
 ## How to use
 
 Checkout the individual scripts and either call them manually or automatically, e.g. in CI jobs
+Find all the dependencies which are required in [dependencies.yaml](https://github.com/os-autoinst/scripts/blob/master/dependencies.yaml]. You get all of them installed with one package.
+
+```bash
+sudo zypper in os-autoinst-scripts-deps
+```
```

Comment:
I would keep it short. There's no reason to explain how to use `zypper` here.

```suggestion
Required dependencies are listed in
[dependencies.yaml](https://github.com/os-autoinst/scripts/blob/master/dependencies.yaml].
You can install all dependencies under openSUSE via the `os-autoinst-scripts-deps` package.
```

Reply:
it is not to explain zypper. I would like to keep it. it is easier when you look up to get started and easier when you forget that particular thing, as opposed to read through the text. I hope it is not a problem.